cx/spark/server.js

```diff
@@ -58,13 +59,40 @@ async function init(){
   registerWebhook();
 }
 
+// Converts Spark message to a detectIntent request. 
+function sparkToDetectIntent(query, sessionPath){
+     const request = {
+     session: sessionPath,
+     queryInput: {
+         text: {
+             text: query,
```

Comment:
From the function name and description, looks like this function should accept a Spark request and convert it to DetectIntentRequest, rather than just accepting any query text?